test/E2E/Helpers.js

```diff
@@ -0,0 +1,34 @@
+exports._retrieveJQuery = page => () =>
```

Comment:
The formatting throughout this module is difficult to read

Reply:
We're going to add JS formatting/linting to CI, I'll do that soon and you can merge it in